js/util/ajax.js

```diff
@@ -68,7 +77,6 @@ exports.getImage = function(url, callback) {
             context.drawImage(img, 0, 0);
             return context.getImageData(0, 0, img.width, img.height).data;
         };
-        return img;
```

Comment:
It looks like these code changes originate in https://github.com/mapbox/mapbox-gl-js/pull/3302. Can you remove that code from this PR and keep the two independent?